Resources/Audio/Items/Whistle/attributions.yml

```diff
@@ -0,0 +1,34 @@
+- files: ["whistle_1.ogg"]
+  license: "CC0-1.0"
+  copyright: "User strongbot on freesound.org. Modified by Fahasor on github"
+  source: "https://freesound.org/people/strongbot/sounds/568995/"
+
+- files: ["whistle_2.ogg"]
+  license: "CC0-1.0"
+  copyright: "User strongbot on freesound.org. Modified by Fahasor on github"
+  source: "https://freesound.org/people/strongbot/sounds/568995/"
+  
+- files: ["whistle_3.ogg"]
+  license: "CC0-1.0"
+  copyright: "User strongbot on freesound.org. Modified by Fahasor on github"
+  source: "https://freesound.org/people/strongbot/sounds/568995/"
+  
+- files: ["whistle_4.ogg"]
+  license: "CC0-1.0"
+  copyright: "User strongbot on freesound.org. Modified by Fahasor on github"
+  source: "https://freesound.org/people/strongbot/sounds/568995/"
+  
+- files: ["whistle_5.ogg"]
+  license: "CC0-1.0"
+  copyright: "User strongbot on freesound.org. Modified by Fahasor on github"
+  source: "https://freesound.org/people/strongbot/sounds/568995/"
+  
+- files: ["whistle_6.ogg"]
+  license: "CC0-1.0"
+  copyright: "User strongbot on freesound.org. Modified by Fahasor on github"
+  source: "https://freesound.org/people/strongbot/sounds/568995/"
+  
+- files: ["whistle_7.ogg"]
+  license: "CC0-1.0"
+  copyright: "User strongbot on freesound.org. Modified by Fahasor on github"
+  source: "https://freesound.org/people/strongbot/sounds/568995/"
```

Comment:
files is a list so just use the 1 attribution as this level of duplication is unnecessary.